karmada-CLI-installtion-and-deploy-example/intro.md

```diff
@@ -0,0 +1,39 @@
+### background：
+
+1. The config, config-member1, and config-member2 files are in the $HOME/. kube/directory are kubeconfig files for the Karmada cluster, member1 cluster, and member2 cluster, respectively
```

Comment:
The kubeconfig files for the Karmada cluster, member1 cluster, and member2 cluster are located in the `$HOME/.kube` directory. These files are named config, config-member1, and config-member2 respectively.